github/github_test.go

```diff
@@ -25,6 +25,7 @@ import (
 
 	gogithub "github.com/google/go-github/v56/github"
 	"github.com/stretchr/testify/require"
+	"k8s.io/utils/ptr"
```

Comment:
Nit: move this import along with other `*.k8s.io` imports.

Reply:
Done.